packages/components/organisms/f-cookie-banner/src/components/CookieBanner.vue

```diff
@@ -511,38 +514,64 @@ export default {
 }
 
 .c-cookieBanner-content {
-    margin: 0 auto;
-    padding: f.spacing(f);
-    text-align: left;
+    margin-block: 0;
+    margin-inline: auto;
+    padding-block: 0;
+    padding-inline: f.spacing(d);
+    text-align: start;
+    overflow-y: auto;
+
+    @include f.media('>=mid') {
+        padding-inline: f.spacing(f);
+    }
+
+    @include f.media('<mid') {
+        background:
+            // Scroll shadow cover (hides the shadow when you scroll down)
+            linear-gradient(
+                rgba(f.$color-white, 0),
+                f.$color-white 70%
+            ) center bottom,
+
+            // Scroll shadow
+            radial-gradient(
+            farthest-side at 50% 100%,
+                rgba(f.$color-black, 0.3),
```

Comment:
do we have a token for the opacity modifier? might be worth checking the elevation tokens

Reply:
I've tried using the elevation tokens for this (which include colour and opacity) but I can't get it to work, so I'm not gonna spend too much more time on this